### DISCUSSION TITLE
My minio started as a single disk, but the storage was erasure block storage

### DISCUSSION THREAD

**Discussion by RunInto** [minio/minio#15831] [Q&A] [answered]
I deleted the ~/.minio/ and the.minio.sys/ under the bucket, and restarted it. It's still erasureproof block storage.

I want to single disk, not block run, how to do?

**Comment by vadmeste** [comment]
@RunInto it is still single disk without data redunduncy. 

The new format supports much more S3 features. The old format is deprecated for new deployments but still supports old deployments.

**Comment by RunInto** [comment]
@vadmeste How does the new format of SNSD mode ensure high availability of data by utilizing the underlying storage volume? please.

**Comment by klauspost** [comment, marked as answer]
@RunInto It does not. It merely stores metadata and other file information in a similar format. But if your drive dies you lose all your data.

It will however allow you to use all features like versioning, locking, etc.

You should not use SNSD in production unless you, yourself can guarantee the storage will never fail.

**Comment by ravindk89** [comment]
https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-single-node-single-drive.html

This functionality has been in place for around 4 months now. We deprecated the legacy-style FS backend format in favor of a consistent erasure code backend. 

You can use `mc` to perform S3 operations using bash-like semantics if you still need to access the object storage resource.
